pkg/transformers/greenmask/greenmask_integer_transformer_test.go

```diff
@@ -0,0 +1,415 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package greenmask
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/xataio/pgstream/pkg/transformers"
+)
+
+func TestNewIntegerTransformer(t *testing.T) {
+	tests := []struct {
+		name        string
+		generator   transformers.GeneratorType
+		params      transformers.Parameters
+		wantErr     bool
```

Comment:
It's usually easier to have a `wantErr` of type `error` (instead of a boolean and a string) that you can then check and validate against nil and not nil values by using the golang [`errors`](https://pkg.go.dev/errors) utilities, like `errors.ErrorIs`. It simplifies the test cases, and also keeps the logic for validation below simple.
`errors.ErrorIs` is often also used throughout the code to catch errors of a particular type from different packages without having to know the specifics of the error message. 

In this case, you can check that the error returned fits the invalid parameters type and simplify the test body. Example: https://github.com/xataio/pgstream/blob/main/pkg/transformers/greenmask/greenmask_string_transformer_test.go#L69